aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/AbstractTest.xtend

```diff
@@ -58,11 +56,11 @@ abstract class AbstractTest extends AbstractXtextTests {
 		}
 		return setup.createInjectorAndDoEMFRegistration
 	}
-	
+
 	/**
 	 * Subclasses may add modules here, they will be mixed-in.
-	 */	
-	protected def void collectModules(List<Module> modules) {
+	 */
+	protected def void collectModules(List<? extends Module> modules) {
```

Comment:
This typing is incorrect - Java does not allow adding elements to a list of that type. It must be `List<Module>`.